app/ui/src/app/import-export/import-export.module.ts

```diff
@@ -0,0 +1,41 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Routes } from '@angular/router';
+import { FileUploadModule } from 'ng2-file-upload';
+import { SyndesisCommonModule, PatternflyUIModule } from '@syndesis/ui/common';
+
+import { ImportModule, ImportIntegrationComponent } from './import';
+import { ImportExportComponent } from './import-export.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    component: ImportExportComponent,
+    children: [
+      {
+        path: 'import',
+        component: ImportIntegrationComponent
+      },
+      {
+        path: '',
+        redirectTo: 'import-export'
+      }
+    ]
+  }
+];
```

Comment:
It' usually a good practice to promote routing to its own routing module, and import it inot the gneeral domain module. Check the Metrics module for an example and download the John Papa's Angular Snippets for autogenerating that code in your IDE